Game/Scripts/UIGameManager.cpp

```diff
@@ -67,24 +90,23 @@ void UIGameManager::Update(float deltaTime)
 
 void UIGameManager::MenuIsOpen()
 {
-	if (menuIsOpen == false)
-	{
-		mainMenuObject->Disable();
-		hudCanvasObject->Enable();
-		player->SetMouse(false);
-	}
-
-	if (menuIsOpen == true)
+	if (menuIsOpen)
 	{
 		mainMenuObject->Enable();
 		hudCanvasObject->Disable();
 		player->SetMouse(true);
```

Comment:
```suggestion
		player->SetMouse(menuIsOpen);
```
you can put this outside of the if-else block and you will have one less line in your code 😄